package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-textarea-autosize",
   "description": "textarea component for React which grows with content",
-  "version": "5.2.0",
+  "version": "5.2.1",
```

Comment:
updating `dist` files and the version is part of the publishing process, should not be done in PRs